docs/usage.rst

```diff
@@ -986,3 +986,12 @@ for e.g. regular pruning.
 Further protections can be implemented, but are outside of Borgs scope. For example,
 file system snapshots or wrapping ``borg serve`` to set special permissions or ACLs on
 new data files.
+
+SSH batch mode
+~~~~~~~~~~~~~~
+
+When running |project_name| using an automated script, ``ssh`` might still ask for a password,
+even if there is an SSH key for the target server. To make scripts more robust, set the
```

Comment:
SSH would also ask for a password when public key access is disabled serverside. The server negotiates the allowed authenticaton methods, so the server admin could enable TOTP 2FA on the server or disable public key access (not sure why one would do that one though) without warning if you don't control the server's ssh settings (i.e. you are not the owner).

It's not that important of an option to use (because it's only used in unattended scripts, and only necessary when the SSH authentication requirements might change/increase) but it's still a good measure in scripts.